content/logs/_index.md

```diff
@@ -2,320 +2,178 @@
 title: Log Management
 kind: Documentation
 description: "Configure your Datadog Agent to gather logs from your host, containers & services."
+further_reading:
+- link: "logs/graph"
+  tag: "Documentation"
+  text: "Perform analytics with Log Graphs"
+- link: "logs/processing"
+  tag: "Documentation"
+  text: Learn how to process your logs
+- link: "logs/parsing"
+  tag: "Documentation"
+  text: Learn more about parsing
 ---
 
 {{< vimeo 243374392 >}}
 
-## Getting started with the Agent
+## Log Collection
 
-Log collection requires an Agent version >= 6.0. Older versions of the Agent do not include the `Log collection` interface that is used for log collection.
+Log collection is the beginning of your journey in the wonderful world of log-management. Use either the [Datadog Agent][6] if you want to forward logs directly [from your hosts](#from-your-hosts) or your [containerized environments](#from-a-docker-environment), or use our [AWS Lambda function](#from-aws-services) if you want to monitor directly your AWS services logs.
+If you are already using a log-shipper daemon, refer to our dedicated documentation for [Rsyslog][1], [Syslog-ng][2], [NXlog][3], [FluentD][4], and [Logstash][5].
```

Comment:
The new line seems to not be taken into account in the preview: https://cl.ly/3z3p0d08140P